proto/wtools/abase/l0/l1.test/Long.test.s

```diff
@@ -8754,6 +8754,238 @@ function longRight( test )
   test.shouldThrowErrorSync( () => _.longRight( [ 1, 2 ], 1, 0, ( e ) => e, () => 1 ) );
 }
 
+//
+
+function performance( test )
+{
+   /*
+    Average of 10 runs of 50 million iteration of each test case
+    Values below are in seconds
+
+    |      **Routine**       | **Njs : v16.1.0** |**Njs : v14.16.1** |
```

Comment:
no need information per type. avarage time does only sense. make it simpler.

Reply:
/*
    Average of 10 runs of 50 million iteration of each test case
    Values below are in seconds
    |-------------------|-------------------|
    | **Njs : v16.1.0** |**Njs : v14.16.1** |
    | :---------------: |:---------------:  |
    | 11.3262481598487  | 13.4397384615385  |            |
    |-------------------|-------------------|
  */